Automattic/wp-calypso
# WooCommerce: Setup/NUX: Add manage fledgling UX

Work in progress

See #13613

#### How it Works
- Depending on whether the merchant has completed setup or not, we will either render the "Howdy" (egg) or the "Now that your store setup is complete." (but no orders yet) (fledgling) view
- Those containers then render the widgets for each of those views
- There is a intermediate container, widget-group, that dynamically handles rendering full width, half-width, or third-width children using flexbox tricks
- This PR includes the parts of the momma view shared with fledgling too

#### To test:

- First, npm run test-client client/extensions/woocommerce/state
- Navigate to http://calypso.localhost:3000/store/:site
- Make sure you see the “Howdy! Let's set up your store & start selling” view aka the egg view
- Note: The “I’m finished” button in the egg view is not hooked up yet (next PR)
- Use the https://developer.wordpress.com/docs/api/console/ to POST to /sites/{siteID}/calypso-preferences/woocommerce setting finished_initial_setup to 1
- Refresh the page
- Make sure you see the “Now that your setup is complete it\'s time to spread the word” view aka the fledgling view
- Make sure each sharing link works in the share widget
- Make sure you see the three latest articles from the “Start your store” category on woocommerce.com in the reading widget
- Make sure clicking in the email address field expands the footer of the reading widget
- Make sure the subscribe button is not enabled until a valid email address and at least one role is picked
- Note: The “subscribe” button in the reading widget is not hooked up yet (next PR)
- Make sure the example order widget appears
- Note: There is no example order to display yet, so clicking that button doesn’t do much good yet
- Make sure the test and view my site widget appears
- Make sure clicking on the view my site widget button opens a new tab to your site
- Lastly, to see the new components in the momma view, force `storeHasOrders` to true in `render` in woocommerce/app/dashboard/manage.js

cc @kellychoffman @jameskoster
